app/src/main/java/com/vincentengelsoftware/androidimagecompare/OverlaySlideActivity.java

```diff
@@ -34,8 +34,11 @@ protected void onCreate(Bundle savedInstanceState) {
             VesImageInterface image_front = findViewById(R.id.overlay_slide_image_view_front);
             Bitmap bitmapSource = Images.second.getAdjustedBitmap();
 
+
+            ImageButton hideShow = findViewById(R.id.overlay_transparent_button_hide_front_image);
+
             SeekBar seekBar = findViewById(R.id.overlay_slide_seek_bar);
-            SlideHelper.addSeekbarLogic(seekBar, image_front, leftToRight, bitmapSource);
+            SlideHelper.addSeekbarLogic(seekBar, image_front, leftToRight, bitmapSource, hideShow);
             seekBar.setProgress(50);
 
```

Comment:
With the Code below added here in line 43, it will be the correct icon after you change phone rotation.

```
ImageButton swapDirection = findViewById(R.id.overlay_slide_button_swap_seekbar);
if (leftToRight.value) {
    swapDirection.setImageResource(R.drawable.ic_slide_ltr_vector);
} else {
    swapDirection.setImageResource(R.drawable.ic_slide_rtl_vector);
}
SlideHelper.setSwapSlideDirectionOnClick(
        swapDirection,
        seekBar,
        leftToRight
);
```